pagai/services/database_explorer.py

```diff
@@ -36,11 +29,18 @@ def handle_between_filter(col, value):
 }
 
 MSSQL = "MSSQL"
+ORACLE11 = "ORACLE11"
```

Comment:
We'll need to enable ORACLE11 in pyrog, right?

Reply:
indeed